regression/checkpr.sh

```diff
@@ -230,10 +233,14 @@ case $target in
     ;;
 
   # Moonlight: Fulldiagnostics (Debug)
-  ml-fey*) startCI ${project} Debug fulldiagnostics $pr ;;
+  ml-fey*) startCI ${project} Debug $pr ;;
 
   # Snow: Debug
-  sn-fe*) startCI ${project} Debug na $pr ;;
+  sn-fe*)
+    startCI ${project} Debug na $pr
```

Comment:
Debug and fulldiagnostics have different logic paths in the code.  We might be able to remove a Debug or Release build from Moonlight instead.  For now, I will leave this as is.